base_location_geonames_import/data/country.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
It should be better to call this `res_country_data.xml`, but I'm not blocking anymore.